test/extensions/filters/common/ext_authz/ext_authz_http_impl_test.cc

```diff
@@ -346,11 +353,16 @@ using HeaderValuePair = std::pair<const Http::LowerCaseString, const std::string
 // Verify client response headers when authorization_headers_to_add is configured.
 TEST_F(ExtAuthzHttpClientTest, AuthorizationOkWithAddedAuthzHeaders) {
   const auto expected_headers = TestCommon::makeHeaderValueOption(
-      {{":status", "200", false}, {"x-downstream-ok", "1", false}, {"x-upstream-ok", "1", false}});
+      {{":status", "200", envoy::config::core::v3::HeaderValueOption::OVERWRITE_IF_EXISTS_OR_ADD},
+       {"x-downstream-ok", "1",
+        envoy::config::core::v3::HeaderValueOption::APPEND_IF_EXISTS_OR_ADD},
+       {"x-upstream-ok", "1",
+        envoy::config::core::v3::HeaderValueOption::OVERWRITE_IF_EXISTS_OR_ADD}});
   const auto authz_response = TestCommon::makeAuthzResponse(
       CheckStatus::OK, Http::Code::OK, EMPTY_STRING, TestCommon::makeHeaderValueOption({}),
-      // By default, the value of envoy.config.core.v3.HeaderValueOption.append is true.
```

Comment:
FTR I think this comment wasn't actually true (cc @dio ?) since the code was checking for `option.append().value()` without using the PROTOBUF_GET_... wrapper to default to true.

Reply:
Yes that's true @esmet